docs/data-sources/migration.md

```diff
@@ -27,6 +27,7 @@ data "atlas_migration" "hello" {
 - `cloud` (Block, Optional) (see [below for nested schema](#nestedblock--cloud))
 - `config` (String) The configuration file for the migration
 - `dir` (String) Select migration directory using URL format
+- `env_name` (String) The name of the environment used for reporting runs to Atlas Cloud. Default: tf
```

Comment:
```suggestion
- `env_name` (String, Optional) The name of the environment used for reporting runs to Atlas Cloud. Default: tf
```

It’s optional, right?

Reply:
It's listing under the optional attributes list, ;)

Reply:
<img width="521" alt="Screenshot 2024-09-15 at 18 09 13" src="https://github.com/user-attachments/assets/edba3a00-724a-4c6e-a835-e39c4250511f">